.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.1","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/rkashyap/Library/Application Support/IntelliJIdeaLI15/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```

Comment:
Two general comments regarding sbt and gradle.
1. The compute model code relies on generated code from PDL files via gradle so we should add documentation on that and the gradle build.
2. Let's ensure that the existing sbt build functions properly and that further development is not blocked.